content/working_with/official_plugins/Infrastructure/vcloud.md

```diff
@@ -10,407 +10,706 @@ aliases:
     - /developer/official_plugins/vcloud/
 ---
 
-The vCloud plugin enables you to use a vCloud-based infrastructure for deploying services and applications.
+The vCloud Plugin 2 enables you to use a vCloud-based infrastructure for deploying services and applications.
```

Comment:
what do you mean by:`Plugin 2`?

Reply:
It is v2.X and replaces the Tosca plugin which is v1.X.